src/rust/engine/options/src/parse.rs

```diff
@@ -331,28 +332,22 @@ impl Parseable for String {
     }
 }
 
-pub(crate) trait ListMember: Parseable + DeserializeOwned {}
-
-impl ListMember for bool {}
-
-impl ListMember for i64 {}
-
-impl ListMember for f64 {}
-
-impl ListMember for String {}
-
 // If the corresponding unexpanded value points to a @fromfile, then the
 // first component is the path to that file, and the second is the value from the file,
 // or None if the file doesn't exist and the @?fromfile syntax was used.
 //
 // Otherwise, the first component is None and the second is the original value.
 type ExpandedValue = (Option<PathBuf>, Option<String>);
 
+// fn mk_parse_err(err: impl Display, path_opt: Option<Path>) -> ParseError {
+//     let error_src = if let Some(path) = path_opt { format!("reading {path}") } else { "parsing value" };
+//     ParseError::new(format!("Problem {error_src} for {{name}}: {err}"))
+// }
+
```

Comment:
Is this commented out code intended to remain?

Reply:
Ooops, deleted.